README.md

```diff
@@ -35,6 +35,10 @@ This variable is required. It supports one of the following values:
 
 - `clean`: Remove snapshots that conform to the specified prefix and pattern
 
+- `revert`: Revert to snapshots that are specifed by either the pattern or set.  If either the source LV or
+            snapshot are open, the merge is deferred until the next time the server reboots and the
+            source logical volume is activated.
```

Comment:
Is it common for users to want to reboot the system after doing this?
Will users know that the snapshot is open, and that a reboot is required for the merge to apply?

We have other roles that require a reboot for the changes to take effect.
https://github.com/linux-system-roles/kernel_settings#kernel_settings_reboot_ok
and
https://github.com/linux-system-roles/kernel_settings#variables-exported-by-the-role

basically, if the role needs to reboot for a change to take effect, if you set `$ROLENAME_reboot_ok: true`, then the role will reboot the machine.  Otherwise, the role will set `$ROLENAME_reboot_required` to `true` to let the user know that a reboot is required for the changes to take effect.

Reply:
> Is it common for users to want to reboot the system after doing this? Will users know that the snapshot is open, and that a reboot is required for the merge to apply?

Yes, I think it is common to want a reboot.  But, some users might want to avoid a reboot.

If users are merging/reverting to LVM snapshots, I think they would know.  We should reiterate the rules in the role documentation.

> 
> We have other roles that require a reboot for the changes to take effect. https://github.com/linux-system-roles/kernel_settings#kernel_settings_reboot_ok and https://github.com/linux-system-roles/kernel_settings#variables-exported-by-the-role
> 
> basically, if the role needs to reboot for a change to take effect, if you set `$ROLENAME_reboot_ok: true`, then the role will reboot the machine. Otherwise, the role will set `$ROLENAME_reboot_required` to `true` to let the user know that a reboot is required for the changes to take effect.

I opened an issue to track : https://github.com/linux-system-roles/snapshot/issues/16